services/api/tests/test_connected_agent_auth.py

```
@@ -499,3 +505,20 @@ async def test_keyring_isolation_accepts_the_trusted_jobos_mcp_config(tmp_path:
     config = (tmp_path / "codex-home" / "config.toml").read_text(encoding="utf-8")
     assert "[mcp_servers.jobos]" in config
     assert str(launcher.resolve()) in config
+
+
+def test_support_diagnostics_are_bounded_and_credential_free() -> None:
+    secret = "credential-that-must-not-enter-support-output"
```

Ruff (0.16.2)

[error] Possible hardcoded password assigned to: "secret"

(S105)